mlflow/langchain/langchain_tracer.py

```diff
@@ -36,41 +37,33 @@ class MlflowLangchainTracer(BaseCallbackHandler, metaclass=ExceptionSafeAbstract
     input arguments added to original function call.
 
     Args:
-        parent_span: Optional parent span for the trace. If provided, spans will be
-            created under the given parent span. Otherwise, a single trace will be
-            created. Example usage:
-
-            .. code-block:: python
-
-                from mlflow.langchain.langchain_tracer import MlflowLangchainTracer
-                from langchain_community.chat_models import ChatDatabricks
-
-                chat_model = ChatDatabricks(endpoint="databricks-llama-2-70b-chat")
-                with mlflow.start_span("Custom root span") as root_span:
-                    chat_model.invoke(
-                        "What is MLflow?",
-                        config={"callbacks": [MlflowLangchainTracer(root_span)]},
-                    )
         prediction_context: Optional prediction context object to be set for the
             thread-local context. Occasionally this has to be passed manually because
             the callback may be invoked asynchronously and Langchain doesn't correctly
             propagate the thread-local context.
+        set_span_in_context: If True, the span created by this callback will be set as
```

Comment:
Is this for the users who want to keep the existing behavior? Just to confirm, since I don't see the downside of always setting the span in the context.

Reply:
Good question. This is for handling an edge cases for `RunnableSequence.batch()`, where LangChain implementation is super peculiar and we cannot set span in the context for tracing that particular API.